File: week-07/day-3/07.js
'use strict';

var numbers = [2, 5, 11, 29];

// create a function that takes an array of numbers and returns a boolean
// it should return true if all the elements are prime, false otherwise

function inspector(array){
    return array.every(function(number){
        for (var i = 2; i <= Math.sqrt(number); i++) {
           if (number % i === 0) {
               return false;
           }
        }
        return true;
    });
}

console.log(inspector(numbers));

// function primeMaker(array){
//     return array.every(function(element){
//         for(var i = 2; i < element; i++){
//             if (element % i === 0){
//                 return false;
//             }
//         } return true;
//     })
// }
//
// console.log(primeMaker(numbers));

File: week-03/day-3/07.py
# create a function that takes a list and returns a new list with all the elements doubled

def doubleList():
    myList = [10, 20, 30, 40]
    doubledList = []
    a = 0

    while a < len(myList):
        position = a
        doubledList.append(myList[position]*2)
        a += 1

    print("The doubled list: ", doubledList)

doubleList()
